test/bdd/fixtures/agent-wasm/.env

```diff
@@ -11,6 +11,21 @@
 
 # Agent configurations
 USER_WASM_IMAGE=docker.pkg.github.com/trustbloc/edge-agent/user-agent-wasm
+AGENT_REST_IMAGE=aries-framework-go/agent-rest
+AGENT_REST_IMAGE_TAG=latest
```

Comment:
Better to have a specific tag than `latest`.

Reply:
Added `amd64-0.1.3`. I am not able to use `0.1.4-snapshot-fa7a392`. Any idea why?